graviti/dataframe/column/series.py

```diff
@@ -43,20 +43,16 @@ class Series:
     """
 
     schema: PortexType
+    _data: PagingList
+    name: Optional[str]
```

Comment:
the commit type should be `feat`, because stop some feature also effect the features.